diff -ur '--exclude=tmp' '--exclude=log' mia.orig/app/helpers/help_helper.rb mia/app/helpers/help_helper.rb
--- mia.orig/app/helpers/help_helper.rb	2014-06-19 14:52:31.000000000 +0000
+++ mia/app/helpers/help_helper.rb	2022-09-22 17:02:59.008564201 +0000
@@ -7,7 +7,8 @@
   # key out of ct.yml and reutrns a link to that page.
   #
   def link_to_help_page(key, text = 'Help', opts = {})
-    link_to text, (url_for_help_index(key) || HELP_CONTENTS_URL), {:class => :help}.merge(opts)
+    #link_to text, (url_for_help_index(key) || HELP_CONTENTS_URL), {:class => :help}.merge(opts)
+    text
   end
 
   private
