diff -ur a/app/controllers/help_controller.rb b/app/controllers/help_controller.rb
--- a/app/controllers/help_controller.rb	2022-11-04 11:20:58.840989274 +0000
+++ b/app/controllers/help_controller.rb	2022-11-04 12:26:08.917827391 +0000
@@ -13,6 +13,8 @@
   def self.ct_lookup(key)
     @ct ||= YAML.load_file("#{RAILS_ROOT}/help/ct.yml")
     return @ct[key]
+  rescue
+    '/help/contents'
   end
 
 
