diff -u a/fake-gmond.rb b/fake-gmond.rb
--- a/fake-gmond.rb	2013-11-28 13:30:15.000000000 +0000
+++ b/fake-gmond.rb	2022-09-29 12:28:04.226236226 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/ruby
+#!/usr/bin/env ruby
 
 require 'socket'
 
diff -ur a/random.rb b/random.rb
--- a/random.rb 2013-11-28 13:30:15.000000000 +0000
+++ b/random.rb 2022-11-03 18:40:50.570178410 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/ruby
+#!/usr/bin/env ruby
 #
 
 lower=ARGV[0].to_f
diff -ur a/re-pre-heat.rb b/re-pre-heat.rb
--- a/re-pre-heat.rb    2013-11-28 13:30:15.000000000 +0000
+++ b/re-pre-heat.rb    2022-11-03 18:41:03.610079729 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/ruby
+#!/usr/bin/env ruby
 
 require 'rubygems'
 require 'romance'
diff -ur a/sine_wave.rb b/sine_wave.rb
--- a/sine_wave.rb      2013-11-28 13:30:15.000000000 +0000
+++ b/sine_wave.rb      2022-11-03 18:39:25.982815964 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/ruby
+#!/usr/bin/env ruby
 # generates a sine wave value
 # node_number : this is the position of the server in the rack
 # max_node_count : total nodes we are generating numbers for (max size of rack)
