diff -ur a/lib/interchange/interchange.rb b/lib/interchange/interchange.rb
--- a/lib/interchange/interchange.rb	2013-10-10 12:25:30.000000000 +0000
+++ b/lib/interchange/interchange.rb	2022-11-04 14:28:03.395162992 +0000
@@ -42,8 +42,8 @@
     
     def set_start_time
       t = Time.now
-      Log.debug "Storing start time #{t.to_f.inspect} (#{t.inspect})"
-      r = set(START_TIME_KEY, t.to_f)
+      Log.debug "Storing start time #{t.to_i.inspect} (#{t.inspect})"
+      r = set(START_TIME_KEY, t.to_i)
     end
 
     def get_start_time
