diff -ur a/libexec/martha-daemon.rb b/libexec/martha-daemon.rb
--- a/libexec/martha-daemon.rb	2022-10-25 10:27:44.159132376 +0000
+++ b/libexec/martha-daemon.rb	2022-10-25 10:26:51.562130742 +0000
@@ -39,6 +39,13 @@
     Timeout.timeout(60) do
       options = {"load_per_worker" => @load_per_worker, "max_cores" => @max_cores, "simple" => 0}
       eui = File.read("/sys/class/net/eth1/address").chomp.upcase rescue ''
+
+      # HACK: The demo version of MIA only has a single NIC.  This hack allows
+      # matching the correct appliance in memcache `hacor:appliance_configs`.
+      # It works fine as long as a single MIA/ISLA appliance is on the network.
+      eui = 'UNKNOWN' if eui == ''
+      DaemonKit.logger.info("eui set to #{eui}")
+
       Martha.aggregations = nc.process(eui, options, gc, output) unless eui == ''
       # bomb out and clean up workers if we've left some behind for any reason
       if `pgrep martha | wc -l`.to_i > 1
